pkg/backtest/exchange.go

```diff
@@ -293,7 +293,7 @@ func (e *Exchange) QueryWithdrawHistory(ctx context.Context, asset string, since
 	return nil, nil
 }
 
-func (e *Exchange) matchingBook(symbol string) (*SimplePriceMatching, bool) {
+func (e *Exchange) MatchingBook(symbol string) (*SimplePriceMatching, bool) {
```

Comment:
you don't need to expose this method to public?

Reply:
done